src/modules/partners/partner_test.ts

```diff
@@ -78,3 +78,92 @@ describe('tests partners', () => {
       })
   })
 })
+
+// eslint-disable-next-line max-lines-per-function
+describe('test partner suggestion', () => {
```

Comment:
It's good that one describes one thing, later, I'll let Jest group them together